functions/api/getUsfmSrcDownload.ts

```diff
@@ -0,0 +1,52 @@
+import { getHeaders, allParamsAreValid } from "functions/shared"
```

Comment:
@rbnswartz  - do you mean just this first line? Cause line 1 isn't calling a separate CF function.  That's just utility code for making sure the query parameters are what I expect.  If you mean however, why is getUsfmSource a cloudflare function instead of done in place in the browser (it is just a fetch call after all), then the answer is access to CF env. bindings (which is the reason all the api calls are through CF functions).  Of which, the primary binding of which we might potentially use in the future for this would be their key/value storage KV for storing responses from Azure blob storage and not having to go to the origin server, but thus far doesn't seem needed.

Reply:
Usually frameworks have a way of being able to inject things at build time based upon what environment it is targeting. Or at least I've seen that in React. Does astro have anything like that?